internal/build/docker_test.go

```diff
@@ -39,7 +39,7 @@ func TestDigestFromSingleStepOutput(t *testing.T) {
 `
 
 	expected := digest.Digest("sha256:11cd0b38bc3ceb958ffb2f9bd70be3fb317ce7d255c8a4c3f4af30e298aa1aab")
-	actual, err := getDigestFromBuildOutput(bytes.NewBuffer([]byte(input)))
+	actual, err := getDigestFromBuildOutput(context.Background(), bytes.NewBuffer([]byte(input)))
```

Comment:
i think there might be a helper method to create contexts with loggers for testing? if not, can we add one?

Reply:
Oooh thanks for pointing that out!